app/models/concerns/user_authentication_concern.rb

```diff
@@ -13,6 +13,18 @@ module UserAuthenticationConcern
     after_create :create_instance_user
 
     include UserOmniauthConcern
+
+    # Overrides `Devise::Models::Validatable`
+    # This disables the devise email validation for system user.
+    def email_required?
```

Comment:
@lowjoel If included in the authentication concern, the method need to be defined here. so that it comes before `Devise:: Validatable` in the method lookup chain.